systests/wsdl_maven/codegen/src/it/settings.xml

```diff
@@ -30,9 +30,11 @@ under the License.
                     <url>@localRepositoryUrl@</url>
                     <releases>
                         <enabled>true</enabled>
+                        <checksumPolicy>ignore</checksumPolicy>
```

Comment:
Why the `checksumPolicy` needs a change?

Reply:
Because maven 4 enforces the checksum by default.  
Checksums are computed at deploy time, not install time, so they are never present in the local repository for artifacts that have been installed during a build.  But this repository is used as a fake remote repositories, for which maven 4 expects checksums by default.  So we just need to switch the default policy to ignore the fact that the checksums are not present.

Reply:
Got it, thank you for clarifying, I didn't know that